File: 8_Kyu_Kata/abbreviate_a_two_word_name/JavaScript/abbreviateATwoWordName.js
// 8 kyu - Abbreviate a Two Word Name  [ ID: 57eadb7ecd143f4c9c0000a3  (abbreviate-a-two-word-name) ]
// URL: https://www.codewars.com/kata/57eadb7ecd143f4c9c0000a3
// Category: undefined  |  Tags: STRINGS | ARRAYS | FUNDAMENTALS
// *****************************************************************************
const abbrevName = name => name.split(" ").map(el => el[0].toUpperCase()).join(".")

File: 8_Kyu_Kata/abbreviate_a_two_word_name/Python/abbreviate_a_two_word_name.py
# 8 kyu - Abbreviate a Two Word Name  [ ID: 57eadb7ecd143f4c9c0000a3  (abbreviate-a-two-word-name) ]
# URL: https://www.codewars.com/kata/57eadb7ecd143f4c9c0000a3
# Category: NONE  |  Tags: STRINGS | ARRAYS | FUNDAMENTALS
# ******************************************************************************
def abbrev_name(name):
    return f"{name[0].upper()}.{name[(name.index(' ')+1)].upper()}"
